src/parsers/manifest/dash/common/__tests__/manifest_bounds_calculator.test.ts

```diff
@@ -104,18 +105,27 @@ describe("DASH parsers - ManifestBoundsCalculator", () => {
       serverTimestampOffset: 7000,
     });
     manifestBoundsCalculator.setLastPosition(3000, 10);
-    expect(manifestBoundsCalculator.getEstimatedMinimumSegmentTime()).toEqual(
+    expect(manifestBoundsCalculator.getEstimatedMinimumSegmentTime(0)).toEqual(
       7 + 5 - 4 - 3,
     );
+    expect(manifestBoundsCalculator.getEstimatedMinimumSegmentTime(4)).toEqual(
+      7 + 5 - 4 - 3 - 4,
+    );
     performanceNow = 25000;
-    expect(manifestBoundsCalculator.getEstimatedMinimumSegmentTime()).toEqual(
+    expect(manifestBoundsCalculator.getEstimatedMinimumSegmentTime(0)).toEqual(
       7 + 25 - 4 - 3,
     );
+    expect(manifestBoundsCalculator.getEstimatedMinimumSegmentTime(10)).toEqual(
+      7 + 25 - 4 - 3 - 10,
+    );
```

Comment:
I know this was already there but I don't really understand where the value come from ?

Reply:
After looking closer, I understood  that 7 is serverTimestampOffset / 1000, 25 is performanceNow / 1000, 
4 is availabilityStartTime and 3 timeShiftBufferDepth, and 10 is the expected value

Reply:
Yes, but `10` is more the given segment duration